specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview/2020-11-01-preview3/communicationserviceschat.json

```diff
@@ -0,0 +1,1589 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication Chat Service",
+    "description": "Azure Communication Chat Service",
+    "contact": {
```

Comment:
@amrElroumy for preview4 let's remove the contact section. It is not necessary anymore.

Reply:
Let's also create a PR for preview4, I have some minor comments about descriptions and stuff on the swagger, but are non-blocking for merge. So I would rather put them in the PR for preview4.

Reply:
Fixed now.